# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/ServiceWorkerContainerChild.cpp
# Commit: 244df402d6eb
# Full Hash: 244df402d6ebc15d6845c4144245511021408835
# Author: Andrew Sutherland <asutherland@asutherland.org>
# Date: 2024-10-24 09:44:34
# Regressor Bug: 1131324
# File Overlap Count: 1
# Description:
#   Bug 1131324 - Expose ServiceWorkerContainer on WorkerNavigator. r=dom-worker-reviewers,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D213725
# ==============================================================================

diff -r 57f70ba5f27e -r 244df402d6eb dom/serviceworkers/ServiceWorkerContainerChild.cpp
--- a/dom/serviceworkers/ServiceWorkerContainerChild.cpp	Thu Oct 24 03:02:39 2024 +0000
+++ b/dom/serviceworkers/ServiceWorkerContainerChild.cpp	Thu Oct 24 03:02:40 2024 +0000
@@ -65,6 +65,7 @@
     return;
   }
   mTeardownStarted = true;
+  Send__delete__(this);
 }
 
 }  // namespace mozilla::dom